{
  "Mayavi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291477"
  ],
  "NLopt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254979"
  ],
  "PyQt4": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248128"
  ],
  "ara": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279553"
  ],
  "blender": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291492"
  ],
  "btrfs-progs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245650"
  ],
  "conda-build": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291503"
  ],
  "condor": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2269253"
  ],
  "csound": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271396"
  ],
  "dionaea": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247008"
  ],
  "docker-compose": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247015"
  ],
  "fedmsg": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291518"
  ],
  "flatpak-module-tools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2252072"
  ],
  "ikiwiki": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291542",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290514"
  ],
  "jc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247039"
  ],
  "khal": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291549"
  ],
  "krita": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2278646"
  ],
  "libprelude": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247193"
  ],
  "libpreludedb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261338"
  ],
  "mailman3": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291557"
  ],
  "openscap": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2260316"
  ],
  "openvswitch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250661"
  ],
  "petsc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250673"
  ],
  "plplot": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272975"
  ],
  "proxmark3": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245823"
  ],
  "pygsl": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250675"
  ],
  "pyliblo": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248131"
  ],
  "python-amico": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244829"
  ],
  "python-anymarkup-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261545"
  ],
  "python-arm-preprocessing": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291617"
  ],
  "python-astunparse": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279984"
  ],
  "python-avocado": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250847"
  ],
  "python-backrefs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246376"
  ],
  "python-datadog": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290567"
  ],
  "python-devicely": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2277745"
  ],
  "python-devtools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280965"
  ],
  "python-dipy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280596"
  ],
  "python-django-auth-ldap": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290343"
  ],
  "python-django4.2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291678"
  ],
  "python-djvulibre": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226179"
  ],
  "python-dns-lexicon": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291680"
  ],
  "python-elephant": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291685"
  ],
  "python-eventlet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290561"
  ],
  "python-flask-basicauth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261560"
  ],
  "python-gbulb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291712"
  ],
  "python-gensim": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261825"
  ],
  "python-google-cloud-monitoring": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280563"
  ],
  "python-hl7": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247023"
  ],
  "python-ibm-vpc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261572"
  ],
  "python-imagehash": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226213"
  ],
  "python-ipdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280968"
  ],
  "python-jnius": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272966"
  ],
  "python-libusb1": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280555"
  ],
  "python-lmdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259530"
  ],
  "python-logutils": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226233"
  ],
  "python-matplotlib-venn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280562"
  ],
  "python-memray": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280960"
  ],
  "python-mglob": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244827"
  ],
  "python-multiprocess": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246347"
  ],
  "python-network-runner": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291791"
  ],
  "python-openpaperwork-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245800"
  ],
  "python-orderedset": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226269"
  ],
  "python-pathvalidate": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259547"
  ],
  "python-pdir2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275052"
  ],
  "python-ppft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291847"
  ],
  "python-precis_i18n": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246305"
  ],
  "python-py-gql": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246126"
  ],
  "python-pyfastnoisesimd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250674"
  ],
  "python-pymilter": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259637"
  ],
  "python-pytn3270": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246275"
  ],
  "python-pyunicorn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291888"
  ],
  "python-qcustomplot-pyqt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291892"
  ],
  "python-recordclass": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265475"
  ],
  "python-ruffus": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259546"
  ],
  "python-scss": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245678"
  ],
  "python-setuptools_scm_git_archive": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2240107"
  ],
  "python-smartcols": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226335"
  ],
  "python-sphinxcontrib-openapi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279996"
  ],
  "python-stompest": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280565"
  ],
  "python-tensordict": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291943"
  ],
  "python-testinfra": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279554"
  ],
  "python-tkrzw": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291947"
  ],
  "python-tweepy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290566"
  ],
  "python-ufo2ft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291961"
  ],
  "python-usort": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271063"
  ],
  "python-vedo": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291965"
  ],
  "python-wcmatch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246283"
  ],
  "python-webpy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256932"
  ],
  "python-wordcloud": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291974"
  ],
  "python-wsaccel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265482"
  ],
  "python-x2go": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291976"
  ],
  "python3-zope-fixers": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246285"
  ],
  "qpid-dispatch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245601"
  ],
  "qtile": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291994"
  ],
  "quodlibet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259638"
  ],
  "rmlint": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272923"
  ],
  "rpkg-util": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276893"
  ],
  "transtats-cli": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264228"
  ],
  "ubertooth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259655"
  ],
  "uwsgi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290556"
  ],
  "vrpn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247291"
  ],
  "wesnoth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280558"
  ],
  "xonsh": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2283616"
  ],
  "znc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2292226"
  ]
}